frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,78 @@
+import { NS } from '../utils/consts';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+// Create storage class from
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+
+// Create new pool form
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
+
+export const poolMessage = {
+  PROGRESS:
+    'The creation of an OCS storage cluster is still in progress or have failed please try again after the storage cluster is ready to use.',
+  POOL_START: (poolName) => `Pool ${poolName} creation in progress`,
+  POOL_DUPLICATED: (poolName) => `Pool "${poolName}" already exists`,
+  POOL_CREATED: (poolName) => `Pool ${poolName} was successfully created`,
+};
+
+export const storagePool = {
+  prepareStorageClassForm: () => {
+    // Select provisioner
+    cy.byTestID('storage-class-provisioner-dropdown')
+      .click()
+      .get('#openshift-storage\\.rbd\\.csi\\.ceph\\.com-link')
+      .click();
+    cy.byTestID('expand-pvc-checkbox').check();
+
+    // Open a storage pool creation form
+    cy.byTestID(storagePoolDropdown).click();
+    cy.byTestID('create-new-pool-button').click();
+  },
+  create: (poolName: string, replicaCount: string, poolCreationJobStatus: string) => {
+    // Make sure the storage pool creation form is open
+    modal.modalTitleShouldContain('Create New Storage Pool');
+    modal.submitShouldBeDisabled();
+    modal.shouldBeOpened();
+
+    cy.byTestID(poolNameTextBox)
+      .clear()
+      .type(poolName);
+    cy.byTestID(replicaDropdown)
+      .click()
+      .byLegacyTestID(replicaCount)
+      .click();
+    modal.submitShouldBeEnabled();
+
+    // Create new pool
+    cy.get('#confirm-action').click();
```

Comment:
Unable to use the submit method from modals. The storage class creation page and the pool creation model both have submit button.

Reply:
If two elements are returned we could chain `first` or `last` and then get the required element.